index.js

```diff
@@ -1,20 +1,44 @@
-let { readFile, stat } = require("fs").promises;
-let path = require("path");
-let { FileFinder } = require("faucet-pipeline-core/lib/util/files/finder");
+let { readFile, stat } = require("node:fs/promises");
+let path = require("node:path");
+let { FileFinder } = require("./lib.js");
+/** @import {
+  *   Config,
+	*   AssetManager,
+	*   FaucetPlugin,
+	*   FaucetPluginOptions,
+	*   FaucetPluginFunc,
+	*   CompactorMap,
+	*   WriteFileOpts,
+	*   ProcessFile
+	*   } from "./types.ts"
+	**/
```

Comment:
This indentation seems really odd?

(Unrelated aside: Multi-line statements is something I struggle with in JSDoc - to the point of grundgingly accepting excessively long lines... )

FIWW: I'd move type imports [to the bottom](https://prepitaph.org/articles/typed-javascript/#fn:ordering) of the file.

Reply:
Moved the type imports to the bottom in 341098f4ff4c62194a7e575828012112cb804b39

Reply:
I don't know what the correct way to inline this is to be honest :joy:

Reply:
Addressed in the latest review.